operator/control-plane/src/aws.rs

```diff
@@ -360,80 +398,69 @@ impl Aws {
                 error!(stderr);
                 return Err(anyhow!(
-                    "Error fetching network interface qdisc configuration: {stderr}"
+                    "Error removing network interface qdisc configuration: {stderr}"
                 ));
             }
-            let entries: Vec<&str> = stdout.trim().split('\n').collect();
-            let mut is_any_rule_set = true;
-            if entries[0].to_lowercase().contains("qdisc mq 0: root") && entries.len() == 1 {
-                is_any_rule_set = false;
-            }
-
-            // remove previously defined rules
-            if is_any_rule_set {
-                let (_, stderr) = Self::ssh_exec(
-                    sess,
-                    &("sudo tc qdisc del dev ".to_owned() + &interface + " root"),
-                )?;
-                if !stderr.is_empty() {
-                    error!(stderr);
-                    return Err(anyhow!(
-                        "Error removing network interface qdisc configuration: {stderr}"
-                    ));
-                }
-            }
+        }
 
-            let (_, stderr) = Self::ssh_exec(
-                sess,
-                &("sudo tc qdisc add dev ".to_owned()
-                    + &interface
-                    + " root tbf rate "
-                    + &bandwidth.to_string()
-                    + "kbit burst 4000Mb latency 100ms"),
-            )?;
+        let (_, stderr) = Self::ssh_exec(
+            sess,
+            // interpolation is safe since values are integers
+            &format!("sudo tc qdisc add dev ens5 root tbf rate {bandwidth}kbit burst 4000Mb latency 100ms"),
+        )?;
 
-            if !stderr.is_empty() {
-                error!(stderr);
-                return Err(anyhow!("Error setting up bandwidth limit: {stderr}"));
-            }
-        } else {
-            return Err(anyhow!("Error fetching network interface name"));
+        if !stderr.is_empty() {
+            error!(stderr);
+            return Err(anyhow!("Error setting up bandwidth limit: {stderr}"));
         }
 
-        let iptables_rules: [&str; 4] = [
+        Ok(())
+    }
+
+    // Goal: set up iptables rules for salmon
+    // first two rules are just expected to be there
+    // rest of the rules are replaced if needed
+    fn run_fragment_iptables_salmon(sess: &Session) -> Result<()> {
+        let iptables_rules: [&str; 5] = [
             "-P PREROUTING ACCEPT",
+            // expected to exist due to how the images are built
+            "-A PREROUTING -m addrtype --dst-type LOCAL -j DOCKER",
             "-A PREROUTING -i ens5 -p tcp -m tcp --dport 80 -j REDIRECT --to-ports 1200",
             "-A PREROUTING -i ens5 -p tcp -m tcp --dport 443 -j REDIRECT --to-ports 1200",
-            "-A PREROUTING -i ens5 -p tcp -m tcp --dport 1025:65535 -j REDIRECT --to-ports 1200",
+            "-A PREROUTING -i ens5 -p tcp -m tcp --dport 1024:65535 -j REDIRECT --to-ports 1200",
```

Comment:
update max port to 61439

Reply:
for salmon we can forward everything since incoming and outgoing connections don't interfere

might still change for consistency